contrib/platform/src/com/sun/jna/platform/win32/Wtsapi32.java

```diff
@@ -86,6 +95,61 @@
 	 */
 	int WTS_SESSION_REMOTE_CONTROL = 0x9;
 
+    /**
+     * Specifies the current server
+     */
+    HANDLE WTS_CURRENT_SERVER_HANDLE = new HANDLE(null);
+
+    /**
+     * Specifies the current session (SessionId)
+     */
+    DWORD WTS_CURRENT_SESSION = new DWORD(-1L);
+
+    /**
+     * Specifies any-session (SessionId)
+     */
+    DWORD WTS_ANY_SESSION = new DWORD(-2L);
+
+    int WTS_PROCESS_INFO_LEVEL_0 = 0;
+    int WTS_PROCESS_INFO_LEVEL_1 = 1;
+
+    /**
+     * Contains extended information about a process running on a Remote Desktop
+     * Session Host (RD Session Host) server. This structure is returned by the
+     * WTSEnumerateProcessesEx function when you set the pLevel parameter to
+     * one.
+     * 
+     * @see <A HREF=
+     *      "https://docs.microsoft.com/en-us/windows/desktop/api/wtsapi32/ns-wtsapi32-_wts_process_info_exa">WTS_PROCESS_INFO_EXA</A>
+     * @see <A HREF=
+     *      "https://docs.microsoft.com/en-us/windows/desktop/api/wtsapi32/ns-wtsapi32-_wts_process_info_exw">WTS_PROCESS_INFO_EXW</A>
+     */
+    @FieldOrder({ "SessionId", "ProcessId", "pProcessName", "pUserSid", "NumberOfThreads", "HandleCount",
+            "PagefileUsage", "PeakPagefileUsage", "WorkingSetSize", "PeakWorkingSetSize", "UserTime", "KernelTime" })
+    class WTS_PROCESS_INFO_EX extends Structure {
+        public DWORD SessionId;
+        public DWORD ProcessId;
+        public Pointer pProcessName; // Either LPSTR or LPWSTR
```

Comment:
This can be mapped as String, if you specify the correct type mapper - see `Sspi#SecPkgInfo` for a sample.

Reply:
Funny, I originally had (and still have) it as a `String` in my project, and it worked, but I didn't know the inner workings well enough to know if it handled the Unicode/ANSI conversions.  I'll put it back!

Reply:
Not sure if you already found the "explanation". If not this is the short version: The default Windows function mapper maps to either the ASCII or the UNICODE version of a function (if they follow the windows conventions) based on the value of the `w32.ascii` system property. The same logic is then applied in `com.sun.jna.win32.W32APITypeMapper.W32APITypeMapper(boolean)`. A string is on the fly converted to a WString and on the return way converted back to a string.